examples/tutorials/advanced/vectors.py

```diff
@@ -72,7 +72,7 @@
 ###############################################################################
 # The default unit of vector length is centimeters,
 # however, this can be changed to inches or points. Note that, in PyGMT,
-# one point is defined as 1/72 inch.
+# one point is defined as 1/72 inches.
```

Comment:
```suggestion
# one point is defined as 1/72 inche.
```
should be singular in my opinion, let's the what our native speakers say 😉

Reply:
I think the "1/72 inch" makes more sense; the correct grammar would be "1/72 of an inch" but I think that is unnecessarily wordy.

Reply:
Thanks @willschlitzer.
Changed it back to the short singular version, see commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/3fb23aecc13bb3ba98eb1bafd35e299cf85f4807.